applications/areg-portal/frontend/src/components/Support/FAQs.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import { makeStyles } from '@mui/styles';
+import { vars } from "../../theme/variables";
+import { AccordionPlusIcon, AccordionMinusIcon } from "../icons";
+import { Box, Grid, Typography, Accordion, AccordionSummary, AccordionDetails } from "@mui/material";
+import { faqsInfo } from "../../utils/faqsInfo";
+const { primaryTextColor, primarySubheaderColor, primaryHeaderColor } = vars;
+
+const useStyles = makeStyles((theme?: any) => ({
+    faqBox: {
```

Comment:
The linting is not consistent. Should be 2 spaces, not 4. One strategy is to use the VSCode ESLint plugin